include/constants/battle_ai.h

```diff
@@ -34,20 +34,20 @@
 #define AI_FLAG_PREFER_BATON_PASS     (1 << 6)
 #define AI_FLAG_DOUBLE_BATTLE         (1 << 7)   // removed, split between AI_FLAG_CHECK_BAD_MOVE & AI_FLAG_CHECK_GOOD_MOVE
 #define AI_FLAG_HP_AWARE              (1 << 8)
+#define AI_FLAG_POWERFUL_STATUS       (1 << 9)   // AI prefers moves that set up field effects or side statuses, even over a kill. Usega with caution
```

Comment:
Use with caution? What does that even mean?

Reply:
It means that it should be used knowingly that it will be used over a move that faints the target. I can remove it if you think it is redundant.

Reply:
I think all AI flags should be used with caution, haha, and your comment imo is clear enough in that it may cause unwanted behaviour